underfs/b2/src/main/java/alluxio/underfs/b2/B2UnderFileSystem.java

```diff
@@ -0,0 +1,286 @@
+      B2StorageClient b2StorageClient, String bucketId, String bucketName, short bucketMode,
+      String accountId) {
+    super(alluxioURI, conf);
+    requireNonNull(alluxioURI, "alluxioURI is null");
+    requireNonNull(conf, "conf is null");
+    requireNonNull(b2StorageClient, "b2StorageClient is null");
+    requireNonNull(bucketId, "bucketId is null");
+    requireNonNull(bucketName, "bucketName is null");
+    requireNonNull(bucketMode, "bucketMode is null");
+    requireNonNull(accountId, "accountId is null");
+
+    LOG.debug("alluxioURI: {}", alluxioURI);
+    LOG.debug("buckedId: {}", bucketId);
+    LOG.debug("bucketName: {}", bucketName);
+    LOG.debug("bucketMode: {}", bucketMode);
+    LOG.debug("accountId: {}", accountId);
+
+    mB2StorageClient = b2StorageClient;
+    mBucketId = bucketId;
+    mBucketName = bucketName;
+    mBucketMode = bucketMode;
+    mAccountId = accountId;
+  }
+
+  @Override public boolean createEmptyObject(String key) {
+    return false;
+  }
+
+  @Override protected OutputStream createObject(String key) throws IOException {
+    return null;
+  }
+
+  @Override protected boolean copyObject(String src, String dst) throws IOException {
+    return false;
+  }
+
+  //
+  @Override protected boolean deleteObject(String key) throws IOException {
+    return false;
+  }
+
+  @Override protected ObjectPermissions getPermissions() {
+    LOG.debug("getPermissions");
+    return new ObjectPermissions(mAccountId, mAccountId, mBucketMode);
+  }
+
+  @Nullable @Override protected ObjectStatus getObjectStatus(String key) throws IOException {
+    LOG.debug("getObjectStatus");
+    LOG.debug("bucketName: {}, key: ", mBucketName, key);
+    try {
+      B2FileVersion fileInfo = mB2StorageClient.getFileInfoByName(mBucketName, key);
+      return new ObjectStatus(fileInfo.getFileName(), fileInfo.getContentSha1(),
+          fileInfo.getContentLength(), fileInfo.getUploadTimestamp());
+    } catch (B2Exception e) {
+      if (e.getCode().equals("not_found")) {
+        LOG.debug("key not found {}", key);
+        return null;
+      }
+      throw new RuntimeException(e);
+    }
+  }
+
+  @Override protected String getFolderSuffix() {
+    LOG.debug("getFolderSuffix {}", FOLDER_SUFFIX);
+    return FOLDER_SUFFIX;
+  }
+
+  @Override protected String getRootKey() {
+    LOG.debug("getRootKey {}", B2_SCHEME + mBucketName);
+    return B2_SCHEME + mBucketName;
+  }
+
+  @Override
+  protected InputStream openObject(String key, OpenOptions options, RetryPolicy retryPolicy)
+      throws IOException {
+    LOG.debug("openObject");
+    LOG.debug("key [{}]", key);
+    LOG.debug("options [{}]", options);
+    try {
+      return new B2InputStream(mBucketName, key, mB2StorageClient, options.getOffset(),
+          options.getLength());
+    } catch (ServiceException e) {
+      throw new IOException(e.getMessage());
+    }
+  }
+
+  @Override public String getUnderFSType() {
+    LOG.debug("getUnderFSType");
```

Comment:
Enabling debug logging on the `UnderFileSystemWithLogging` class should negate any need for these types of log messages

Reply:
I removed this line